dashboard/public/js/main.js

```diff
@@ -103,3 +103,66 @@ function firstClientVisit() {
 		return true;
 	}
 }
+var textBox = document.getElementById("password-box");
+var password = document.getElementById('password');
+
+function passwordStrength() {
+	var parentDiv = document.createElement("div");
+	parentDiv.id = "passtrength-div";
+	
+	var meter = document.createElement("METER");
+	meter.id = "password-strength-meter";
+	meter.max = "4";
+	
+	var strengthText = document.createElement("p");
+	strengthText.id = "password-strength-text";
+
+	parentDiv.appendChild(meter);
+	parentDiv.appendChild(strengthText);
+	password.insertAdjacentElement('afterend', parentDiv);
+	parentDiv.className = "col-md-12";
+
+	var strength = {
+        0: "Poor",
+        1: "Mediocre",
+        2: "Okay",
+        3: "Good",
+        4: "Strong"
+    }
+
+	textBox.addEventListener('input', () => {
+		const val = textBox.value;
+		const result = zxcvbn(val);
+		meter.value = result.score;
+
+			if (val !== "") {
+				strengthText.innerHTML = "Strength: " + "<strong>" + strength[result.score] + "</strong>";
+			} else {
+				strengthText.innerHTML = "";
+			}
+	});
+}
+
+var confirmBox = document.getElementById("confirm-password");
+
+function confirmPassword() {
+	var confirmText = document.createElement("p");
+	confirmText.id = "confirm-text";
+	confirmBox.insertAdjacentElement("afterend", confirmText);
+
+	confirmBox.addEventListener('input', () => {
+		if (confirmBox.value != ""){
+			if (confirmBox.value !== textBox.value){
+				confirmText.innerHTML = "<strong>Passwords don't match!</strong>"
+				$('#submit').prop('disabled', true);
+			}else if (confirmBox.value === textBox.value){
+				$('#submit').prop('disabled', false);
+				confirmText.innerHTML = "<strong>Passwords match!</strong>"
+			}else{
```

Comment:
Try to correct the spacing in the script as much as you can.

Reply:
Also, avoid mixing javascript and jquery selectors in the same function.
I think you can achieve the same behavior with `document.getElementById("submit").disabled = false;`